test/remote/gateways/remote_reach_test.rb

```diff
@@ -191,7 +204,7 @@ def test_failed_capture
     assert_equal 'Not Found', response.message
   end
 
```

Comment:
💭 Opinion:
 Could be nice to have a remote test case to see what's happened if the payment_model is nil.

Reply:
test